src/shared/seccomp-util.c

```diff
@@ -88,3 +88,229 @@ int seccomp_add_secondary_archs(scmp_filter_ctx *c) {
+                "socketpair\0"
+        }, {
+                /* Unusual, obsolete or unimplemented, some unknown even to libseccomp */
+                .set_name = "@obsolete",
+                .value =
+                "_sysctl\0"
+                "afs_syscall\0"
+                "break\0"
+                "fattach\0"
+                "fdetach\0"
+                "ftime\0"
+                "get_kernel_syms\0"
+                "get_mempolicy\0"
+                "getmsg\0"
+                "getpmsg\0"
+                "gtty\0"
+                "isastream\0"
+                "lock\0"
+                "madvise1\0"
+                "modify_ldt\0"
+                "mpx\0"
+                "pciconfig_iobase\0"
+                "perf_event_open\0"
+                "prof\0"
+                "profil\0"
+                "putmsg\0"
+                "putpmsg\0"
+                "query_module\0"
+                "rtas\0"
+                "s390_pci_mmio_read\0"
+                "s390_pci_mmio_write\0"
+                "s390_runtime_instr\0"
+                "security\0"
+                "sgetmask\0"
+                "ssetmask\0"
+                "stty\0"
+                "subpage_prot\0"
+                "switch_endian\0"
+                "sys_debug_setcontext\0"
+                "tuxcall\0"
+                "ulimit\0"
+                "uselib\0"
+                "vm86\0"
+                "vm86old\0"
+                "vserver\0"
+        }, {
+                /* Nice grab-bag of all system calls which need superuser capabilities */
+                .set_name = "@privileged",
+                .value =
+                "@clock\0"
+                "@module\0"
+                "@raw-io\0"
+                "acct\0"
+                "bdflush\0"
+                "bpf\0"
+                "chown32\0"
+                "chown\0"
+                "chroot\0"
+                "fchown32\0"
+                "fchown\0"
+                "fchownat\0"
+                "kexec_file_load\0"
+                "kexec_load\0"
+                "lchown32\0"
+                "lchown\0"
+                "nfsservctl\0"
+                "pivot_root\0"
+                "quotactl\0"
+                "reboot\0"
+                "seccomp\0"
+                "setdomainname\0"
+                "setfsgid32\0"
+                "setfsgid\0"
+                "setfsuid32\0"
+                "setfsuid\0"
+                "setgid32\0"
+                "setgid\0"
+                "setgroups32\0"
+                "setgroups\0"
+                "sethostname\0"
+                "setregid32\0"
+                "setregid\0"
+                "setresgid32\0"
+                "setresgid\0"
+                "setresuid32\0"
+                "setresuid\0"
+                "setreuid32\0"
+                "setreuid\0"
+                "setuid32\0"
+                "setuid\0"
+                "stime\0"
+                "swapoff\0"
+                "swapon\0"
+                "sysctl\0"
+                "syslog\0"
```

Comment:
not everything of this one is really privileged. unprivileged clients may read the kernel log buffer for example. probably shouldn't be listed here...